localisation/00_wc_artifact_localisation.csv

```diff
@@ -134,6 +134,8 @@ mantle_of_gorak_tul;Mantle of Gorak Tul;;Gorak Tuls Mantel;;;;;;;;;;;x
 mantle_of_gorak_tul_desc;A shimmering leather mantle with feathers on the outside that has survived the centuries unscathed thanks to dark magic. It is property of the one true ruler of Kul Tiras.;;Ein schimmernder, lederner Mantel, der die Jahrhunderte dank dunkler Magie unbeschadet �berstanden hat. Eigentum des einzig wahren Herrschers von Kul Tiras.;;;;;;;;;;;x
 lionhorn_of_stormwind;Lion Horn of Stormwind;;L�wenhorn von Stormwind;;;;;;;;;;;x
 lionhorn_of_stormwind_desc;The powerful sound of this horn makes the hearts of the soldiers beat faster.;;Der kraftvolle Klang dieses Horns l�sst die Herzen der Soldaten h�her schlagen.;;;;;;;;;;;x
+justice_gaze;Justice Gaze;;Blick der Gerechtigkeit;;;;;;;;;;;x
+justice_gaze_desc;A masterfully crafted winged helmet made of bronze and steel. May its wearer walk forever in the light.;;Ein meisterhaft gefertigter, gefl�gelter Helm aus Bronze und Stahl. M�ge sein Tr�ger auf ewig im Lichte wandeln.;;;;;;;;;;;x
```

Comment:
```suggestion
justice_gaze_desc;A masterfully crafted winged helmet made of bronze and steel. May its wearer walk forever in the Light.;;Ein meisterhaft gefertigter, gefl�gelter Helm aus Bronze und Stahl. M�ge sein Tr�ger auf ewig im Lichte wandeln.;;;;;;;;;;;x
```

Reply:
And that happens when i make DeepL translate back and forth to check if the alternative looks better. Totally missed this :D